modules/matchRoutes.js

```diff
@@ -160,7 +162,19 @@ function matchRouteDeep(
       }
     }
 
-    const result = getChildRoutes(route, location, onChildRoutes)
+    const progressState = { 
```

Comment:
In most cases the route will not use this object; no reason to do stuff like call `createParams` unless we know we need to call `getChildRoutes`.